src/Dev/TaskRunner.php

```diff
@@ -183,6 +180,38 @@ protected function taskEnabled($class)
         return true;
     }
 
+    protected function canViewTask(string $class): bool
+    {
+        if ($this->canViewAllTasks()) {
+            return true;
+        }
+
+        $reflectionClass = new ReflectionClass($class);
+        if ($reflectionClass->isAbstract()) {
+            return false;
+        }
+
+        $task = Injector::inst()->get($class);
+        if (!$task->hasMethod('canView') || !$task->canView()) {
+            return false;
+        }
+
+        return true;
+    }
+
+    protected function canViewAllTasks(): bool
+    {
+        return (
+            Director::isDev()
+            // We need to ensure that DevelopmentAdminTest can simulate permission failures when running
+            // "dev/tasks" from CLI.
+            || (Director::is_cli() && DevelopmentAdmin::config()->get('allow_all_cli'))
+            || Permission::check("ADMIN")
+            || Permission::check("ALL_DEV_ADMIN")
+            || Permission::check("BUILDTASK_CAN_RUN")
```

Comment:
```suggestion
            || Permission::check(['ADMIN', 'ALL_DEV_ADMIN', 'BUILDTASK_CAN_RUN'])
```
This is a little tidier I think. Please make this change (single quotes, and a single check with an array) to all of these new permission checks.